docs/pages/technical-specs/expo-updates-1.mdx

```diff
@@ -17,37 +17,43 @@ This is the specification for Expo Updates, a protocol for delivering updates to
-2. If a new manifest is downloaded, the client library will proceed to make additional requests to download and store any missing assets specified in the manifest.
+1. The client library will make a [request](#update-request) for the most recent update, with constraints specified in the headers.
+2. If a new update is downloaded, the client library will process its parts:
+    - For a _manifest_ part, the client library will proceed to make additional requests to download and store any missing assets specified in the manifest. This is considered a new _manifest_.
+    - For a _message_ part, the client library will consume the message depending on the message type and update the its local state accordingly.
 3. The client library will edit its local state to reflect that a new update has been added to the local cache. It will also update the local state with the new `expo-manifest-filters` and `expo-server-defined-headers` found in the response [headers](#manifest-response-headers).
 
 The primary consumers of this spec are Expo Application Services and organizations that wish to manage their own update server to satisfy internal requirements.
 
-## Manifest Request
+## Update Request
 
 A conformant client library MUST make a GET request with the headers:
 
-1. `expo-platform`, to specify the platform type the client is running on.
+1. `expo-protocol-version: 1`, to specify version 1 of this expo-updates specicifcation.
+2. `expo-platform`, to specify the platform type the client is running on.
    - iOS MUST be `expo-platform: ios`.
    - Android MUST be `expo-platform: android`.
    - If it is not one of these platforms, the server SHOULD return a 400 or a 404
-2. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file.
-3. Any headers stipulated by a previous responses' [server defined headers](#manifest-response-headers).
+3. `expo-runtime-version` MUST be a runtime version compatible with the client. A runtime version stipulates the native code setup a client is running. It should be set when the client is built. For example, in an iOS client, the value may be set in a plist file.
+4. Any headers stipulated by a previous responses' [server defined headers](#manifest-response-headers).
 
 A conformant client library MUST send at least one of `accept: application/expo+json`, `accept: application/json`, or `accept: multipart/mixed` based on the supported response structures though SHOULD send `accept: application/expo+json, application/json, multipart/mixed`. A conformant client library MAY express preference using "q" parameters as specified in [RFC 7231](https://datatracker.ietf.org/doc/html/rfc7231#section-5.3.1), which default to `1`.
```

Comment:
Not sure what is meant by `based on the supported response structures though SHOULD send` -- can you clarify this?

Reply:
Let's discuss this on Monday to make sure I understand it :)

Reply:
Essentially a compliant client can choose to only support non-multipart for simplicity (for example, if a consumer of the protocol only needs the manifest and doesn't need any of the other stuff). This sentence is saying that it has the option, but it is suggested to support multipart responses since there is more that can be done with those responses.